src/components/SimilarProducts/SimilarProducts.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import Slider from 'react-slick';
+import { FormattedMessage } from 'react-intl';
+import { useHistory } from 'react-router-dom';
+import Product from '../Products/Product';
+import './similarProducts.css';
+
+const SimilarProducts = ({ products = [] }) => {
+  const sliderSettings = {
```

Comment:
If this is static, it's a good idea to move this outside the component, so that it's not assigned again and again for every re-render.

Reply:
I totally missed that, might be a refactoring side effect